README.md

```diff
@@ -40,49 +40,50 @@ extension ContentViewController: Embeddable {}
 
 Furthermore an instance of `EmbeddableContentDelegate` is exposed. This `delegate` can be used to instruct the drawer to perform various tasks by calling the `handle` function on it.
 
-The `handle` function takes an `enum` of type `Drawer.EmbeddedAction` which allows these actions:
+The `handle` function takes an `enum` of type `Drawer.Action` which allows these actions:
 
-- `layoutUpdated(config: Drawer.ContentConfiguration)` to update the layout of your drawer
+- `layoutUpdated(config: Drawer.Configuration)` to update the layout of your drawer
 - `changeState(to: MovementState)` to show/hide your drawer.
 
 After creating the `contentViewController`, initialize an instance of `DrawerCoordinator` in your `backgroundViewController` to initialize the drawer.
 
 ```swift
 let drawer = DrawerCoordinator(contentViewController: contentVC,
-backgroundViewController: self,
-drawerBackgroundType: .withColor(UIColor.black.withAlphaComponent(0.5)))
+                               backgroundViewController: self,
+                               drawerBackgroundType: .withColor(UIColor.black.withAlphaComponent(0.5)))
 ```
 
 #### Displaying a Drawer
-After your content's views have finished creating and you are ready to display the drawer, create an instance of `Drawer.ContentConfiguration` to set the drawer state and properties.
+After your content's views have finished creating and you are ready to display the drawer, create an instance of `Drawer.Configuration` to set the drawer state and properties.
 
 ```swift
-let contentConfiguration = Drawer.ContentConfiguration(duration: animationDuration,
-                                                        embeddedFullHeight: maxHeight,
-                                                        state: .minimised,
-                                                        embeddedMinimumHeight: minHeight,
-                                                        cornerRadius: Drawer.ContentConfiguration.CornerRadius(fullSize: 20,
-                                                                                                                minimised: 0),
-                                                        dismissCompleteCallback:
-{ [weak self] in
-    guard let self = self else { return }
-    //TODO: Drawer dismissed.
-})
+
+let options: [Drawer.Configuration.Key : Any] = [
+.animationDuration: 0.5,
+.fullHeight: maxHeight,
+.minimunHeight: minHeight,
```

Comment:
Same typo, probably won't be caught when you rename the property so mentioning it here again